### ISSUE TITLE
Turbine won't accept steam

### ISSUE THREAD

**Issue by lrdfang** [ZeroNoRyouki/BigReactors#55]
I can't get a turbine to accept steam while playing on the DW20 1.10 pack. The reactor works fine and is generating steam but it has a broken texture and the turbine will not accept any so it is not working.

![2016-12-10_19 04 20](https://cloud.githubusercontent.com/assets/19766690/21077058/b057a8a6-bf0b-11e6-82b3-a735a9f3b2d9.png)
![2016-12-10_19 04 24](https://cloud.githubusercontent.com/assets/19766690/21077059/b0593fae-bf0b-11e6-815b-b1a03f29eb38.png)
![2016-12-10_19 04 26](https://cloud.githubusercontent.com/assets/19766690/21077060/b0593c16-bf0b-11e6-9d47-5084dd74dde4.png)

**Comment by RogueLogic** [comment]
Fist, as with your other report, please update to latest version if possible.
Second, what do the ports covered by the nodes look like?
Third, this would seem to be a problem with Extra Utilities, if your setup is done correctly, as the turbine works fine with EnderIO conduits as seen below.
![2016-12-11_00 03 36](https://cloud.githubusercontent.com/assets/9971693/21079153/236e2c12-bf3d-11e6-97cf-6a25ad7ca6e6.png)

**Comment by ZeroNoRyouki** [comment]
Where is the broken texture?

Is the reactor steam tank always full or do you see the level changing?

**Comment by lrdfang** [comment]
I found the problem, and the source of the broken textures. The problem was caused by railcraft steam. That is the source of the broken texture, the reactor was full of steam and was outputting to the transfer node fine. I even tried it with enderio pipes, tanks, and dimensional transceivers. I even went into a creative word and tried you creative steam generator and it would not fill the turbines steam tank. When I realized the broken texture was with railcraft not extreme reactors I was able to disable it in the config and it restored function.

I haven't played with any other steam sources so I am unsure if the issue was caused by RC not registering their steam properly or if turbines just didn't like it.

**Comment by ZeroNoRyouki** [comment]
I'm using whichever "steam" fluid is already registered in the system (or register my own if no steam is around). So, was your problem only texture-related? Is the turbine working now (spinning, producing energy, etc) ?

**Comment by lrdfang** [comment]
It is working now that I have disabled railcraft's steam. By default the DW20 pack ended up with forge using railcrafts steam. That was the reason for the broken texture. Along with that while using the railcraft steam the turbine would not accept any steam in, spin up or operate at all. I even tried using the creative steam generator from your mod and it wouldn't accept it. 

After I disabled the railcraft steam registration forge switched to I believe IC2 steam and everything started working. I have steam input, the turbines spin up everything is now good. Due to this it seems that the issue is due cross-mod issue between rail-craft and Extreme reactors.
